.travis.yml

```diff
@@ -1,21 +1,20 @@
 ---
 language: python
-python: "2.7.15"
+python: "3.8.6"
 sudo: required
 
 env:
-  - ansible_version: 2.4.5
-  - ansible_version: 2.7.9
-  - ansible_version: 2.8.1
+  - ansible_version: 2.8.8
 
 services:
   - docker
 
 install:
-  - pip install ansible==${ansible_version}
-  - pip install -r test-requirements.txt
+  - pip3 install pipenv
+  - pipenv install ansible==${ansible_version}
```

Comment:
if you add ansibles version on your requirements file, you can omit this line.

Reply:
In fact, in order to do more reproducible tests, a better workflow would be:
1. update test-requirements
2. generate Pipfile and Pipfile.lock (i.e. running pipenv install -r test-requirements.txt)
3. commit Pipfile and Pipfile.lock
So the sequence in travis would be:
install:
  - pip install  pipenv
  - pipenv sync